### PR TITLE
Holding a security knife actually makes you faster.

### PR DESCRIPTION
**Pull request by Irradiation**
Seriously, the description of the knife even said so and nobody actually coded this? Fucking lazy c*ders who hate REALISM. Also comes with a side of April Fools for TRUE realism.

[consistency][qol][bugfix][tested]
:cl:
 * rscadd: Spessmen just now have remembered that holding a security knife makes them run faster.

### CODE REVIEW

**Comment by MadmanMartian** [comment]
[User has been banned for this post]

**Comment by Irradiation** [comment]
No fun allowed

**Comment by ShiftyRail** [comment]
RED ONES GO FASTAH

File: code/modules/mob/living/carbon/human/human_movement.dm [hunk 1]
```diff
@@ -56,7 +56,15 @@
 
 /mob/living/carbon/human/movement_tally_multiplier()
 	. = ..()
-
+	if(locate(/obj/item/weapon/kitchen/utensil/knife/tactical) in held_items)
```
Comment (ShiftyRail) [inline code/modules/mob/living/carbon/human/human_movement.dm]: Not to be too much of a party pooper, but a `locate` call in the middle of the main movement proc might cause some overhead and I'm pretty certain that a 1/1000 difference won't be noticeable

Comment (Irradiation) [reply code/modules/mob/living/carbon/human/human_movement.dm]: I asked around if there was a better thing to do than use an isType() check and I got told to do this by Exxion. Shrug.

Comment (Exxion) [reply code/modules/mob/living/carbon/human/human_movement.dm]: You didn't say you were doing it in the main moveloop

**Comment by Tokanova** [comment]
I like it

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
The number 0.999 seems to imply that this is a joke, but this actually is like a 33% change if I'm not mistaken

Which one is it?

**Comment by Irradiation** [comment]
If you look at stuff like Nuka Cola in Human_movement.dm, they give you a 0.8 modifier which translates to a 20% increase. Unless I'm still sleep-deprived and my own testing was false, this (0.999) should only be a 0.1% speed bonus.

**Comment by ShiftyRail** [comment]
I think that this is below the game's "precision"

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
https://github.com/vgstation-coders/vgstation13/blob/d9e848094b6925abf2f76e05e25d483a69edf908/__DEFINES/game.dm#L10
It's all there in the docs
Did you like uhh test it

**Comment by Irradiation** [comment]
Then it'll just be called a feature :^)

**Comment by dylanstrategie** [comment]
I'm not sure I want to see security officers running around with knives out to go everywhere tbh, that's Goon tier shit

**Comment by SirSkeleto** [comment]
its like youve never even played counter strike

**Comment by Intigracy** [comment]
Individually slowdowns act similar within a range, but as the comment says, combined with stuff like shoe speed increase or hardsuit slowdown, you end up with a difference.

**Comment by Exxion** [comment]
Yeah but in this case it pretty much never will
For this to change your actual speed, your slowdown without it would have to be almost exactly equal to one of the thresholds at which your speed changes

**Pull request merged**